surrogates/youtube-iframe-api.js

```diff
@@ -31,8 +31,13 @@
     // Mappings between the "real" video elements and their placeholder
     // elements.
     const videoElementsByID = new Map();
-    const videoElementByPlaceholderElement = new WeakMap();
-    const placeholderElementByVideoElement = new WeakMap();
+    const videoElementByPlaceholderElement = new Map();
```

Comment:
We need to a strong reference to the video elements now, since they are no longer kept in the DOM. (Also, the behaviour of two WeakMaps pointing to each other's values like this isn't clear. The values of a WeakMap are strongly held references, so I suspect the change from WeakMap -> Map won't change much anyway... but better to be sure!)